music_assistant/providers/soundcloud/__init__.py

```diff
@@ -215,42 +205,45 @@ async def get_library_playlists(self) -> AsyncGenerator[Playlist, None]:
     async def get_library_tracks(self) -> AsyncGenerator[Track, None]:
         """Retrieve library tracks from Soundcloud."""
         time_start = time.time()
-        async for item in self._soundcloud.get_tracks_liked():
-            track = await self._soundcloud.get_track_details(item)
+        async for track in self._soundcloud.get_track_details_liked(self._user_id):
             try:
-                yield await self._parse_track(track[0])
-            except IndexError:
-                continue
-            except (KeyError, TypeError, InvalidDataError) as error:
-                self.logger.debug("Parse track failed: %s", track, exc_info=error)
+                yield await self._parse_track(track)
+            except (KeyError, TypeError, InvalidDataError, IndexError) as error:
+                # somehow certain track id's don't exist (anymore)
+                self.logger.debug(
+                    "Parse track with id %s failed: %s",
+                    track["id"],
+                    track,
+                    exc_info=error,
+                )
```

Comment:
```suggestion
                self.logger.debug(
                    "%s: Parse track with id %s failed: %s",
                    error.__name__,
                    track["id"],
                    track,
                )
```

Not need to print the full stack trace in this case